show/main.py

```diff
@@ -2806,5 +2806,41 @@ def features():
         body.append([key, status_data[key]['status']])
     click.echo(tabulate(body, header))
 
+#
+# 'container' group
```

Comment:
Provide example after. E.g.,

`# 'container' group (show container ...)`

This helps the reader understand the nesting better.

Reply:
Added this example.